src/sentry/static/sentry/app/views/usageStats/UsageStatsLastMin.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {DataCategory, Organization} from 'app/types';
+
+import {Outcome, UsageSeries} from './types';
+import {formatUsageWithUnits, getFormatUsageOptions} from './utils';
+
+type Props = {
+  organization: Organization;
+  dataCategory: DataCategory;
+  dataCategoryName: string;
+} & AsyncComponent['props'];
+
+type State = {
+  orgStats: UsageSeries | undefined;
+} & AsyncComponent['state'];
+
+/**
+ * Making 1 extra API call to display this number isn't very efficient.
+ * The other approach would be to fetch the data in UsageStatsOrg with 1min
+ * interval and roll it up on the frontend, but that adds unnecessary
+ * complexity and it's gnarly to fetch 90 days of data at 1min intervals.
```

Comment:
And super expensive, you'd likely not be able to fetch that many results as 90 days of 1 min will go past the 10K resultset that snuba will return.